Part_2/UIandFriends/Admin.java

```diff
@@ -0,0 +1,137 @@
+package Final_Project;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.regex.Pattern;
+
+public class Admin {
+	
+	public String checkFormatFlight(Flight toCheck){
+		String toReturn="";
+		boolean badFormat = false;
+		
+		
+		String curr = toCheck.getDest();
+		if(curr.length() > 20 || curr.isEmpty() || curr.contains("_")){
+			badFormat = true;
+			String temp = toReturn;
+			toReturn = temp + " Destination Location format Error, please ensure field is not empty and does not excede 20 characters_";

```

Comment:
This can just be `toReturn += " Destination Location format Error, please ensure field is not empty and does not excede 20 characters"

Underscore proooobably doesn't need to be there